### ISSUE TITLE
Google Play Services version

### ISSUE THREAD

**Issue by battika** [OneSignal/OneSignal-Cordova-SDK#297]
Hello OneSignal Team,

Can you guys please consider updating the Google Play Services version to the latest v11.8.0 (at the time of writing) in the gradle build file? 

It is causing me issues as one of my Cordova plugins not using this method requires 11.8.0 and my application fails to build because of that. Obviously, I can fork the code and change it but it might help others too.

Thank you for your consideration.

**Comment by mxkeke** [comment]
+1 same problem

**Comment by fishgrind** [comment]
same problem here! Please change the version to 11.8.+

**Comment by Creawebs** [comment]
Same problem here, causing issues with cordova google maps

**Comment by fishgrind** [comment]
@Creawebs if you are still having trouble with maps, for now you can fix it:

`\platforms\android\project.properties`

change the following:

`cordova.system.library.5=com.google.android.gms:play-services-maps:11.8.+
cordova.system.library.6=com.google.android.gms:play-services-location:11.8.+
`
and in 

`\platforms\android\onesignal-cordova-plugin\poseidon-build-extras-onesignal.gradle`

change the following:

`def versionGroupAligns = [
    // ### Google Play Services library
    'com.google.android.gms': [
        'version': '11.8.+'
    ],

    // ### Google Firebase library
    // Although not used by OneSignal Firebase has some dependencies on gms
    // If present, ensuring they are aligned
    'com.google.firebase': [
        'version': '11.8.+'
    ],`

basically change the versions to 11.8.+

**Comment by seme1** [comment]
I also have the same problem with googlemaps plugin. Would be appreciated if OneSignal can update google play services.

**Comment by cklinx** [comment]
+1 same problem

**Comment by itzamnamx** [comment]
+1 same issue. And causes issues with other plugins wich uses a different version of GMS

**Comment by itzamnamx** [comment]
I have sent a PR for this issue https://github.com/OneSignal/OneSignal-Cordova-SDK/pull/320

**Comment by jkasten2** [comment]
The pull above has been merged, the next release will include this fix

**Comment by TitikshaDaga** [comment]
I am facing the same issue and I am not bale to find file : \platforms\android\onesignal-cordova-plugin\poseidon-build-extras-onesignal.gradle
and in \platforms\android\onesignal-cordova-plugin\android-build-extras-onesignal.gradle I am not able to find the following lines of code :
def versionGroupAligns = [ // ### Google Play Services library 'com.google.android.gms': [ 'version': '11.8.+' ],

// ### Google Firebase library // Although not used by OneSignal Firebase has some dependencies on gms // If present, ensuring they are aligned 'com.google.firebase': [ 'version': '11.8.+' ],

**Comment by hrhosni** [comment]
@TitikshaDaga That's because the latest onesignal cordova plugin has a different <appname>-build-extras-onesignal.gradle file.
To get the previous one, uninstall onesignal plugin and install a previous version of it:

`cordova plugin add onesignal-cordova-plugin@2.3.1` for example

Copy the contents of the said file and then reinstall latest plugin and paste it. Then upgrade com.google.android.gms and com.google.firebase versions.
